### DISCUSSION TITLE
Android NDK

### DISCUSSION THREAD

**Discussion by naim94a** [zed-industries/zed#13017] [Feature Requests]
I'm working on a C++ for Android codebase and was wondering how to configure Zed to work nicely with the Android's NDK. It seems that even if I get Zed to use `clangd --query-driver=/path/to/ndk/clang` it still resolves system include files instead of using the NDK's sysroot.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
